CRM/Relatedpermissions/Upgrader.php

```diff
@@ -96,4 +102,17 @@ public function upgrade_1502() {
     return TRUE;
   }
 
+  /**
+   * See https://github.com/civicrm/civicrm-core/pull/23865
+   *
+   * @return array
+   */
+  private function getPermissionedRelationshipOptions() {
+    return [
+      CRM_Contact_BAO_Relationship::NONE => ts('None'),
+      CRM_Contact_BAO_Relationship::VIEW => ts('View only'),
+      CRM_Contact_BAO_Relationship::EDIT => ts('View and update')
+    ];
```

Comment:
So this preserves the old definition removed by https://github.com/civicrm/civicrm-core/pull/23865/files#diff-d8f3ea846fa5593a3eec2f1e0c755a5bdade3580b5db78ff81a779340326a6e9  Would a better option be to call the new `CRM_Core_SelectValues::getPermissionedRelationshipOptions()` and reformat to 'id' => 'label' ?

Reply:
@aydun I don't really mind but I don't think it makes much difference? I defined it in the extension so we're not relying on a core function that might change again. @eileenmcnaughton what do you think?

Reply:
I think I agree with keeping a copy in the extension - it's not really a supported function to call from outside core. I like the fact it's private - nice & clear no-one else uses it

Reply:
Fine